Statistics/Histograms/Histogram.cs

```diff
@@ -30,6 +30,7 @@ public class Histogram: IHistogram, IReportMessage
         private const string _type = "Histogram";
         #endregion
         #region Properties
+        public string note { get; set; }//for debugging
```

Comment:
If this was meant to just be temporary for debugging, we should probably remove it as not to add unnecessary complexity to our class.